### ISSUE TITLE
Allow money transfert without creating an expense

### ISSUE THREAD

**Issue by Massedil** [koelleChristian/trickytripper#39]
Now, if someone give money to someone else not linked to something bought, I have to create a expense "A => B" in the "Other" category.

But in the report table, it will be noted as an expense from A paid by B.

Is there a good way to add a money transfer or a refunding ?

**Comment by koelleChristian** [comment]
No, according to your speak, there is only a bad way right now.

**Comment by Marenz** [comment]
I'd add my voice to this feature request. This is very useful when someone wants to pay back some money before/while/after the trip without actually causing a new expense on the total trip costs.

**Comment by Marenz** [comment]
I am thinking of two ways this could be done in the interface

* A checkbox "don't add to the total expense"
* A special category "Not part of the expense"

Of course the phrasing should be worked on :>
